### DISCUSSION TITLE
Error when including a PDF File

### DISCUSSION THREAD

**Discussion by albertovdasilva** [microsoft-search/pnp-modern-search#1434] [Q&A]
When I include a PDF file on the SP list where the search results is getting its value from, I get the error message below. 

[HTTP]:500 - [CorrelationId]:32c4fb9f-80b7-1000-2d0d-297a143587f5 [Version]:16.0.0.21813

Why would this be happening? If I remove the PDF the error goes away.

![PDF Error](https://user-images.githubusercontent.com/68698081/138529329-945199ce-492c-4034-bbd0-9170d535236a.png)

**Comment by wobba** [comment]
Hard to say as we don't know your setup in terms of web part version, provider, query template, or ux template. You could hit F12 and look at the response coming back on the API call and see if there are more information coming back.
